test/webhook.template.yaml

```diff
@@ -110,8 +112,22 @@ webhooks:
       - operations: [ "CREATE", "UPDATE" ]
         apiGroups: ["*"]
         apiVersions: ["*"]
-        resources: ["pods", "deployments", "replicasets", "daemonsets", "jobs", "cronjobs"]
+        resources: ["pods", "deployments", "replicasets", "daemonsets", "jobs", "cronjobs", "ingresses"]
     failurePolicy: Fail
     namespaceSelector:
       matchLabels:
         webhook: enabled
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: test-admin
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-admin
```

Comment:
I know this is for testing purposes, but it also partly serves as a documentation on how to setup the webhook. Having said that,  is `cluster-admin` really needed? I'd prefer the RBAC config to only allow the webhook to list ingresses in all namespaces.

Also, please add a comment stating that this is only needed when ingress validation is enabled.

Reply:
custome role added to the manifest